### DISCUSSION TITLE
Program Q&A

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#4] [Q&A]
Please use this discussion if you have any general questions about the **MathWorks Challenge Projects Program**.

**Comment by jeneus** [comment]
1. Is there a hard deadline for the registration and submission of each challenge project? I don't want to tackle a project if it turns out I only have a month to work on it.
2. What is the criteria used to assess project submissions?

**Reply by robertogl** [reply]
Hi @Jeneus,

Sorry for the late reply. 

1. We provide a four month deadline for each project from the time of registration. However, it is not a hard deadline. You can request an extension by simply filling out a short form that you will receive in an email. We do it to keep track of the people who are still interested in working on our projects out of the many who sign up for them.
2. We check if the submission works as expected and if MATLAB and/or Simulink are properly used. We expect the solution to follow the project description requirements but we are open to consider a solution valid even if it contains variations.

**Comment by ghost** [comment]
Hi! I am interested in working on the solar panel project. As far as I have seen the deadline was on Jan 2024. So am I too late to enroll and work on this project? 
If not, will there be any certification of completion of this project if so what is the requirement. Please let me know!

**Comment by sayaan1808** [comment]
Hi, I am a student at NIT Warangal from India and interested in working and collaborating on Intelligent Trip Planning for Battery Electric Vehicles Using Real-Time Map Data project.

with regards
